### PR TITLE
Fix: Challenge 2 and 3 copy and file name changes

### LINKED ISSUE #619: Update Ch.3 Challenge 2 with HoloCat Pool. Also change names of Challenges 2 and 3.
**UPDATE**
Here's what we need for this issue:

- [ ] Change name of Challenge 2 to Pool vs. BitRey (including URL, challenge copy, and chapter titles)
- [ ] Change name of Challenge 3 to Coop vs. BitRey (including URL, challenge copy, and chapter titles)
- [ ] We also need to need to add a frame around the cooperators in Challenge 2. But we won't be adding badges yet. Bades will only come into play for Challenge 3.

So this is challenge 2 right now.
Screenshot 2023-09-04 at 8 28 27 AM

Challenge 2 will need the right side updated to the below image:
265672139-f24eed4b-d29d-49fb-aabd-0d622196e1f6

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #619 

I did  have to change the file names and also touch the routing of the new files

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
tACK!

File: content/lessons/chapter-3/coop-1.tsx [hunk 1]
```diff
@@ -5,65 +5,39 @@ import { Title } from 'ui'
...
 import { cssVarThemeChange } from 'lib/themeSelector'
 
 export const metadata = {
   title: 'chapter_three.coop_one.title',
-  image: '/assets/images/chapter-2-intro-1.jpg',
   key: 'CH3COO1',
 }
 
-export default function Coop1({ lang }) {
+export default function Pool1({ lang }) {
```
Comment (afterburn) [inline content/lessons/chapter-3/coop-1.tsx:18]: This is a bit confusing, the component is named Pool1 but the file is named coop-1. If this component is meant to be for pool-1 can we not switch this code with whatever is in pool-1.tsx and update the routing accordingly?

File: content/lessons/chapter-3/coop-2.tsx [hunk 2]
```diff
@@ -1,153 +1,20 @@
...
-  const { account } = useAuthContext()
-  const t = useTranslations(lang)
-  const [step, setStep] = useState(0)
-  const [protagonistsBlockAmount, setProtagonistsBlockAmount] = useState(0)
-  const [antagonistsBlockAmount, setAntagonistsBlockAmount] = useState(0)
-  const [protagonistHash, setProtagonistHash] = useState([0, 0, 0, 0])
-  const [antagonistHash, setAntagonistHash] = useState(0)
-  const [showText, setShowText] = useState(true)
-
+export default function Pool2({ lang }) {
```
Comment (afterburn) [inline content/lessons/chapter-3/coop-2.tsx:15]: Same here

File: content/lessons/chapter-3/pool-2.tsx [hunk 3]
```diff
@@ -1,20 +1,153 @@
...
 export const metadata = {
   title: 'chapter_three.pool_two.title',
+  theme: 'solo-1-theme',
+  secondaryTheme: 'solo-1-secondary-theme',
   key: 'CH3POL2',
 }
 
-export default function Pool2({ lang }) {
-  const saveAndProceed = useSaveAndProceed()
+export default function Coop2({ lang }) {
```
Comment (afterburn) [inline content/lessons/chapter-3/pool-2.tsx:27]: Same here

**Review by afterburn — DISMISSED** [review]
Thanks @0tuedon, code looks good but I left some comments above for file structure/naming of components

**Comment by 0tuedon** [comment]
> Thanks @0tuedon, code looks good but I left some comments above for file structure/naming of components

Alright, thanks Kev,  let me fix that.

**Review by benalleng — APPROVED** [review]
LGTM!

**Comment by ecurrencyhodler** [comment]
tACK!

**Pull request merged**